.travis.yml

```diff
@@ -20,10 +23,11 @@ before_install:
 
 install:
   # the `make test-ci` script runs this command already
-  - if [ "$JOB" != "test" ] && [ "$JOB" != "lint" ]; then yarn install; fi
+  - if [ "$JOB" != "test" ] && [ "$JOB" != "test" ] && [ "$JOB" != "lint" ]; then yarn install; fi
```

Comment:
Kinda confused about this - is this not the same thing again? `[ "$JOB" != "test" ] &&`